From e53f3385e0127fcfbf636ac50900d2ea2b01f13e Mon Sep 17 00:00:00 2001
From: GGG <gggkiller2@gmail.com>
Date: Fri, 26 Jan 2024 21:08:18 -0300
Subject: [PATCH] Fix getMaxCPU segfault.

---
 src/libutil/current-process.cc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/libutil/current-process.cc b/src/libutil/current-process.cc
index 352a6a0fb..9a6a1f120 100644
--- a/src/libutil/current-process.cc
+++ b/src/libutil/current-process.cc
@@ -35,6 +35,9 @@ unsigned int getMaxCPU()

         auto cpuMax = readFile(cpuFile);
         auto cpuMaxParts = tokenizeString<std::vector<std::string>>(cpuMax, " \n");
+        if (cpuMaxParts.size() != 2)
+            return 0;
+
         auto quota = cpuMaxParts[0];
         auto period = cpuMaxParts[1];
         if (quota != "max")
--
2.43.0

